### DISCUSSION TITLE
Autohide taskbar for 2024 G14 OLED variants when running on battery

### DISCUSSION THREAD

**Discussion by mk519** [seerge/g-helper#3095] [Ideas]
Any chance this can be a toggle option in Ghelper? Im usually docked for most of the day, but when I go mobile I take some precaution and auto hide the taskbar via windows settings. I know it only a few clicks, but this would be super helpful. It's a bit of an annoyance when I have to redock and have to enable the task bar again. It's would be something similar to the auto option for refresh rates for unplugged. Thoughts? 

btw - very grateful for this tool!

**Comment by seerge** [comment]
@mk519 hello,

But you can change that in windows quite easily :) I'm not a big fan of stuffing app with things you can achieve somehow else.

If you want you can use [Autohotkey to automate it](https://www.autohotkey.com/boards/viewtopic.php?f=76&t=82538)

**Reply by mk519** [reply]
@seerge no problem, and thanks for the suggestion! :)

**Reply by mk519** [reply]
For anyone curious about another work around, I found a custom tool called Windhawk. You can install a mod called 'click on empty taskbar space' and assign 'auto hide taskbar'. It saves you a few clicks from accessing the option via windows settings, as you only have to  double click on the taskbar.
